CONTRIBUTING.md

```diff
@@ -1,37 +1,39 @@
 Contributors
 ============
 
-Submitting an Issue? See the [Submitting Issues](https://github.com/chocolatey/chocolateygui#submitting-issues) section in the [README](https://github.com/chocolatey/chocolateygui/blob/master/README.md#submitting-issues).
+Submitting an Issue? See the [Submitting Issues](https://github.com/chocolatey/chocolateygui#submitting-issues) section in the README.
 
 The process for contributions is roughly as follows:
 
 ## Prerequisites
 
  * Ensure you have signed the Contributor License Agreement (CLA) - without this we are not able to take contributions that are not trivial.
-  * [Sign the Contributor License Agreement.](https://www.clahub.com/agreements/chocolatey/ChocolateyGUI)
+  * [Sign the Contributor License Agreement.](https://cla-assistant.io/chocolatey/ChocolateyGUI)
   * You must do this for each Chocolatey project that requires it.
   * If you are curious why we would require a CLA, we agree with Julien Ponge - take a look at his [post](https://julien.ponge.org/blog/in-defense-of-contributor-license-agreements/).
 
 ## Contributing Process
 
 ### Get Buyoff Or Find Open Community Issues/Features
 
- * Through GitHub, or through the [mailing list](https://groups.google.com/forum/#!forum/chocolateygui) (preferred), you talk about a feature you would like to see (or a bug), and why it should be in Chocolatey.
-   * If approved through the mailing list, ensure the accompanying GitHub issue is created with information and a link back to the mailing list discussion.
+ * Through [GitHub discussions](https://github.com/chocolatey/ChocolateyGUI/discussions) you talk about a feature you would like to see (or a bug), and why it should be in Chocolatey.
+   * If approved, ensure the accompanying [GitHub issue](https://github.com/chocolatey/ChocolateyGUI/issues) is created with information and a link back to the GitHub discussion.
  * Once you get a nod from one of the [Chocolatey Team](https://github.com/chocolatey?tab=members), you can start on the feature.
  * Alternatively, if a feature is on the issues list with the [community tag](https://github.com/chocolatey/chocolateygui/labels/Community), it is open for a patch. You should comment that you are signing up for it on the issue so someone else doesn't also sign up for the work.
 
 ### Set Up Your Environment
 
  * You create, or update, a fork of chocolatey/chocolateygui under your GitHub account.
- * From there you create a branch named specific to the feature.
+ * Clone it to your computer. `git clone https://github.com/chocolatey/ChocolateyGUI.git`
```

Comment:
This should be a step to clone the fork of the repository, not the original repository, right?

Reply:
Of course. Updated to `https://github.com/your-github/ChocolateyGUI.git`. Will this work?